src/views/Auth.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+    <div id="auth">
+        <router-view></router-view>
+    </div>
+</template>
```

Comment:
Create a folder called `layouts` and move this file. It should be directly under `src` folder